src/styles/blocks/search.css

```diff
@@ -0,0 +1,50 @@
+.search {
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  background-image: url('../../images/Search.svg');
+  background-repeat: no-repeat;
+  font-family: Avenir, serif;
+  font-weight: 300;
+  box-shadow: 0 1px 8px #3d4e611a;
+  border-radius: 4px;
+}
+
+.search--big {
+  height: 70px;
+  background-position: 26px 25px;
+  padding: 25px 0 26px 62px;
+  background-size: 19px 19px;
+  font-size: 16px;
+}
+
+.search--big:hover {
+  box-shadow: 0 4px 4px #00000040;
+}
+
+.search:focus {
+  font-weight: 400;
+  background-color: #f6f6f7;
+  border-radius: 3px;
+}
+
+.search--big:focus {
+  box-shadow: 0 4px 4px #00000040;
+}
+
+.search--small {
+  height: 42px;
+  background-position: 13 15;
```

Comment:
The background-position property is missing its unit of measurement. It should be specified in pixels (e.g., '13px 15px'), percentages, or other valid units.